openapi/components/schemas/LegacySubscription.yaml

```
@@ -0,0 +1,482 @@
+  - items
+properties:
+  id:
+    type: string
+    description: ID of the order.
+    readOnly: true
+    maxLength: 50
+    example: ord_01GYJPRKHBD6ZYHH897QCJMBS4
+  orderType:
+    description: |-
+      Specifies the type of order.
+      An order may be a subscription or a one-time purchase.
+    type: string
+    x-basic: true
+    enum:
+      - subscription-order
+  customerId:
+    $ref: ./CustomerId.yaml
+  renewalReminderTime:
+    description: Date and time when the renewal reminder event triggers.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
+  renewalReminderNumber:
+    description: Number of triggered renewal reminder events.
+    type:
+      - 'integer'
+      - 'null'
+    readOnly: true
+  trialReminderTime:
+    description: Date and time when the trial reminder event triggers.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
+  trialReminderNumber:
+    description: Number of triggered trial reminder events.
+    type:
+      - 'integer'
+      - 'null'
+    readOnly: true
+  abandonReminderTime:
+    description: Date and time when the abandon order reminder event triggers.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
+  abandonReminderNumber:
+    description: Number of abandon order reminder events that are triggered.
+    type:
+      - 'integer'
+      - 'null'
+    readOnly: true
+  organizationId:
+    readOnly: true
+    allOf:
+      - $ref: ./OrganizationId.yaml
+  status:
+    description: |-
+      Status of the subscription service.
+      A subscription starts in the `pending` status, and becomes `active` when the service period begins.
+    type: string
+    readOnly: true
+    enum:
+      - pending
+      - active
+      - abandoned
+      - canceled
+      - churned
+      - paused
+      - voided
+      - completed
+      - trial-ended
+  inTrial:
+    description: Specifies if the subscription is currently in a trial period.
+    type: boolean
+    readOnly: true
+  trial:
+    type: object
+    description: |-
+      Trial details.
+      To use plan defaults do not send the `trial` key, or send a `null` value.
+    properties:
+      enabled:
+        description: |-
+          Specifies if there is a trial for this subscription.
+          Plans without trial prices are free trials.
+        type: boolean
+      endTime:
+        description: |-
+          Time and date when the trial ends.
+          If a trial is enabled on this subscription, a value must be provided.
+        type:
+          - 'string'
+          - 'null'
+        format: date-time
+  isTrialOnly:
+    description: |-
+      Specifies if a subscription ends after a trial period.
+      If this value is `true`, recurring settings are ignored.
+    type: boolean
+    default: false
+  invoiceTimeShift:
+    description: |-
+      Shifts issue time and due time of invoices for this subscription.
```

suggestion: for readability
```suggestion
      Adjusts the issue date and time of renewal invoices for this subscription.
```